### ISSUE TITLE
Fetch multireddit submissions

### ISSUE THREAD

**Issue by papagno** [not-an-aardvark/snoowrap#76]
By reading the documentation I couldn't find a way to fetch submissions for a multireddit. Is it possible with snoowrap (or the APIs)?

**Comment by not-an-aardvark** [comment]
No, it's not possible at the moment. This seems like something that should be added.

As a workaround, you can fetch posts from multiple subreddits at a time using `r.getSubreddit('foo+bar')` (which gets all posts from [/r/foo+bar](https://reddit.com/r/foo+bar)).

**Comment by papagno** [comment]
Ok, thanks. Maybe we could add an alias for that request to `MultiReddit`. Unless you want the library to match exactly the official reddit API.

**Comment by papagno** [comment]
The data structure seems to be the same. Is the data set different? I would go with `/r/foo+bar` because the multireddit API is undocumented.

**Comment by not-an-aardvark** [comment]
I don't think `/r/foo+bar` is documented either. I suspect the multireddit API is roughly the same, but it's possible there are subtle differences. It seems like it would be more intuitive for snoowrap to display what would actually appear on a multireddit rather than a hack like `/r/foo+bar`.

**Comment by papagno** [comment]
It makes sense. I agree.

**Comment by lumberbeard** [comment]
In the meantime, I've been trying to figure out how to go about this best I can utilizing your wrapper.  I think it's a bit cleaner to do the get on "https://www.reddit.com/user/Lapper/m/depthhub/.json" route versus the (foo+bar) route but I can only think of how to do the latter way via your api. How would you do it the former way?

**Comment by not-an-aardvark** [comment]
You could use [`oauthRequest`](https://not-an-aardvark.github.io/snoowrap/snoowrap.html#oauthRequest__anchor) and set the URL manually.

**Comment by BinarySpike** [comment]
The below code is what I'm using to get multireddit submissions.  Beware that this function makes a request per 100 submissions.  Use at your own risk:

```
async function GetMultireddit(user, multireddit, count, options) {
    let posts = []
    let previousAfter = null;
    try {
        for (let i = 0; i < count; i += 100) {
            let amount = (i + 100 > count) ? (count - i) : (100) // query 100 or remainder
            let chunk = await Reddit.oauthRequest({uri: `/user/${user}/m/${multireddit}/.json`,
                                                   method: 'get',
                                                   qs:{limit: amount, after: previousAfter}})
            chunk.forEach((submission, index) => posts.push(submission)) // for each submission, push to posts
            previousAfter = chunk._query.after;
        }
    } catch (err) {
        console.log(err);
    }
}
module.exports.GetMultireddit = GetMultireddit;
```
